cmd/thanos/query.go

```diff
@@ -141,6 +144,17 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Errorf("Address %s is duplicated for --rule flag.", dup)
 		}
 
+		// Check if the YAML configuration of request.logging is correct. Exit early if error.
```

Comment:
Can we have this either in helper? We need to repeat this in most of the commands.

Reply:
I don't think having a helper would help, because even if we handle the error in a function, that still needs to return the error to each of the components, so the current setup would be fine :+1: